hail/python/hailtop/hailctl/dev/deploy/cli.py

```diff
@@ -16,6 +16,8 @@ def init_parser(parser):
                         help="Comma or space-separated list of steps to run.", required=True)
     parser.add_argument("--excluded_steps", "-e", nargs='+', action='append', default=[],
                         help="Comma or space-separated list of steps to forcibly exclude. Use with caution!")
+    parser.add_argument("--extra-config", "-c", nargs="+", action='append', default=[],
+                        help="Comma or space-separate list of key=value pairs to add as extra config parameters.")
```

Comment:
this can't be space separated. `unpack_comma_delimited_inputs` splits on `,`. I suspect you're trying to indicate that you can do `-c x=y -c z=a`. Maybe:

> Extra configuration key-value pairs. Format as: key=value. May specify multiple pairs separated by commas.

Reply:
I don't think this is true. Here's `unpack_comma_delimited_inputs`:

```python
    return [s.strip()
            for steps in inputs
            for step in steps
            for s in step.split(',') if s.strip()]
```

If you have `-s deploy_batch,deploy_ci , test_ci, test_batch_0 -s test_batch_1`, you'll get an `inputs` list of `[['deploy_batch,deploy_ci', ',', 'test_ci,', 'test_batch_0'], 'test_batch_1']`, which will split accordingly:

```python
In [4]: def unpack_comma_delimited_inputs(inputs):
   ...:     return [s.strip()
   ...:             for steps in inputs
   ...:             for step in steps
   ...:             for s in step.split(',') if s.strip()]
   ...:

In [5]: unpack_comma_delimited_inputs([['deploy_batch,deploy_ci', ',', 'test_ci,', 'test_batch_0'], ['test_batch_1']])
   ...:
Out[5]: ['deploy_batch', 'deploy_ci', 'test_ci', 'test_batch_0', 'test_batch_1']

In [6]:
```

Reply:
Certainly non-obvious and I had to run it in ipython to be very sure, but I always space-delimit by dev deploys, not comma

Reply:
I would honestly rather just space delimit everything, the code would be far easier to reason about and the example I showed is very ugly, but that would be breaking

Reply:
I didn't know you could use the same flag multiple times, so I coded this for this case which is probably bad. So I'm okay if you change my code:

`-c x=y,a=b,c=d`

Reply:
I've been using `-e deploy_auth,create_accounts` with dev deploy

Reply:
What do you want me to change it to? What you wrote is consistent with the other dev deploy options, and I would above all prefer that they be consistent with each other.

Reply:
What makes sense to you and is consistent.

Reply:
> What makes sense to you and is the right way to do it.

I'm not sure how to interpret this sentence. I said I'd prefer to stay consistent with the other CLI options, which this PR currently is.